styles/_responsive.scss

```
@@ -27,3 +27,20 @@ $breakpoint-desktop: 1300px;
     @content;
   }
 }
+
+// 중간값 구하는 함수
+@function clampValue($min, $val, $max) {
+  @return min(max($val, $min), $max);
+}
+
+// 태블릿 사이즈일 때 px 값을 화면 크기에 비례하게 변환해주는 함수입니다.
+// 최소값(모바일 사이즈), 최대값(데스크톱 사이즈), 둘의 차이(정수)를 입력해주면 변환 완료.
+// @TODO: 세번째 패러미터를 지우고 싶은데 어카면 좋을까여...?
+// 현재는 min, max 값이 모두 픽셀이라 (max-min) 값을 vw 값에 곱해주는 데에 에러가 발생.
+@function responsive-px($min, $max, $diff) {
+  @return clampValue(
+    $min,
+    calc($min + (100vw - $breakpoint-tablet) / 542 * $diff),
+    $max
+  );
+}
```

이건 디자이너 요구사항인가요?? 꼭 필요하다고 하신 게 아니면 저렇게 화면 사이즈에 따라 글자 크기가 자연스럽게 필요하는 코드가 필요한지 잘 모르겠어서요... 그냥 데스크탑 UI / 태블릿 UI / 모바일 UI 경계만 변하고 글자 크기는 각 UI마다 고정하는 건 어떤가요?? 그리고 저랑 형석이도 태블릿 UI에서 글자 크기는 고정하여 작업했는데 어느쪽으로든 통일을 시켜야할 듯 합니다.

만약 저 함수가 꼭 필요하다 가정하면 diff를 따로 넣어주는 건 보기가 좀 안 좋은데 차라리 아래처럼 바꿔주는 게 좋을 것 같아요
```suggestion
@function responsive-px($min, $max) {
  @return clampValue(
    ${min}px,
    calc(${min}px + (100vw - $breakpoint-tablet) / (542 * ($max - $mix) / 1px)),
    ${max}px
  );
}
```

---

아 리뷰 내용을 잘못 알아들었네요 넵 태블릿 구간 고정픽셀로 바꾸겠습니다
태블릿 ui에서 사이즈 1.5배 한 거는 임의로 설정하신 거죠? 혹시 논의된 바가 있나 해서요

---

ㄴㄴ 그냥 제 눈까리로 판단했습니다
근데 em으로 해두면 나중에 배율 바꾸는 건 엄청 쉬워서 다 완성되고 채원이랑 용재님한테 피드백 달라하면 될듯